config/jobs/kubernetes/sig-release/release-branch-jobs/1.17.yaml

```diff
@@ -130,7 +130,7 @@ periodics:
       resources: {}
 - annotations:
     fork-per-release-generic-suffix: "true"
-    testgrid-dashboards: sig-node-kubelet, sig-release-1.17-all
+    testgrid-dashboards: sig-node-kubelet, sig-release-job-config-errors
     testgrid-tab-name: node-kubelet-features-beta
```

Comment:
Can we save this one?
This one is in a good state (same for previous releases - 1.16, 1.15, etc) https://testgrid.k8s.io/sig-release-1.17-all#node-kubelet-features-beta
And it runs some useful tests `--focus="\[NodeFeature:.+\]" --skip="\[Flaky\]|\[Serial\]"`

Reply:
by "save" i mean move it over to blocking/informing

Reply:
@alejandrox1 -- nice catch! Fixed in 1d4af084eb40674bee45e6ddef0655f9ad3812eb.